README.md

```diff
@@ -5,3 +5,30 @@ implementation, and reference Knative client libraries.
 
 If you are interested in contributing, see the Knative community [contribution
 guide](https://github.com/knative/docs/blob/master/community/CONTRIBUTING.md).
+
+
+## How to build
+
+**Dependencies:**
+
+[go mod](https://github.com/golang/go/wiki/Modules#quick-start) is used and required for dependencies
+
+**Requirements:**
+
+  - Golang `1.11.5`
+
+**Building:**
+
+```sh
+$ go build ./cmd/...
+```
+
+**Notes:**
+
+knative CLI must be built outside of the $GOPATH folder, or else you'll encounter a module build error. 
```

Comment:
I would change that part. It is possible to have `knative/client` inside the GOPATH — and this affects, so far, only go 1.11.

To build a "go module enabled" project from inside the GOPATH, you need to export `GO111MODULE` : `export GO111MODULE=on` to force Go 1.11 to use go modules. The default value is being `auto`.